worker/echo/server.go

```diff
@@ -55,3 +55,32 @@ func (s *echoServer) Send(ctx context.Context, d *pb.Data) (*pb.Receipt, error)
 	// Respond to the start request that the work was accepted.
 	return &pb.Receipt{}, nil
 }
+
+// Disconnect implements the "Disconnect" method of the Worker gRPC service.
+func (s *echoServer) Disconnect(ctx context.Context, in *pb.Empty, opts ...grpc.CallOption) (*pb.DisconnectResponse, error) {
+	go func() {
+		log.Infof("received worker disconnect request")
+
+		// Dial the Dispatcher and call "Finish"
+		conn, err := grpc.Dial(yggdDispatchSocketAddr, grpc.WithInsecure())
+		if err != nil {
+			log.Fatal(err)
+		}
+		defer conn.Close()
+
+		// Create a client of the Dispatch service
+
+		c := pb.NewWorkerClient(conn)
```

Comment:
`WorkerClient` represents worker and using it in a worker is not correct, especially using connection to yggdrasil (which does not implement `Disconnect` operation).
IMO the whole `Disconnect` function in the echo worker should be a NOOP (and at most log information it received a Disconnect command).